src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/direct_query_table.scss

```diff
@@ -0,0 +1,15 @@
+.customized-table thead {
+  display: none;
+}
+
+[id^="row"][id$="expansion"] > td:first-child > div:first-child {
+  padding: 0;
+}
+
+.customized-row:first-child > td {
+  border-top: 0;
+}
+
+.customized-row:last-child > td {
+  border-bottom: 0;
+}
```

Comment:
`customized-row` is too generic a name. Please make it a tiny bit more specific to this component. Also `[id^="row"][id$="expansion"]` has a wide blast radius; please wrap it inside the classname of the table.